README.md

```diff
@@ -1,7 +1,8 @@
 ﻿<!-- markdownlint-disable-next-line -->
+
```

Comment:
Hey @Vedant-Manjrekar, Please remove this space. Other than that, I think everything looks good.

Let me know if you require any assistance!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix